### PR TITLE
Fix blocking I/O

### PR DESCRIPTION
**Pull request by riobits**
The `writeFile` function helps prevent blocking I/O. Please test the code because I didn't test it.

### CODE REVIEW

**Comment by riobits** [comment]
Error handling should be managed inside a try/catch block. Documentation needs an update.
1. Error handling
2. Explain the `deleteFile` function

File: src/runtime/server/utils/storage.ts [hunk 1]
```diff
@@ -25,14 +28,11 @@ export const storeFileLocally = (
...
+	await writeFile(`~/${location}${filelocation}/${filename}.${ext}`, binaryString)
+
 	return `${filename}.${ext}`
 }
 
 export const deleteFile = async (filename: string, filelocation: string = '') => {
-	rm(`${location}${filelocation}/${filename}`, () => {
-		console.error(`${filename} does not exist`)
-	})
+	await rm(`${location}${filelocation}/${filename}`)
```
Comment (NyllRE) [inline src/runtime/server/utils/storage.ts:37]: I wonder the decision for removing the file deletion error

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "nuxt-nitro-storage",
-  "version": "0.1.2",
+  "version": "0.1.3",
```
Comment (NyllRE) [inline package.json:3]: the versioning is automatically done with npm so no need to change it manually

**Review by NyllRE — COMMENTED** [review]
great improvements! just wondering about some of the decisions made

File: package.json (hunk 2 above)
Comment (riobits) [reply package.json]: Didn't know that

**Pull request merged**